src/main/resources/application.properties

```
@@ -32,7 +32,7 @@ spring.mail.properties.mail.smtp.starttls.enable=true
 email.sender.from=nao-responda@projeto-mercado.dedyn.io
 
 # "Interruptor" para ligar/desligar o envio de e-mail de confirmacao
-email.confirmation.enabled=false
+email.confirmation.enabled=true
```

Deixar falso por padrao
```suggestion
email.confirmation.enabled=false
```